registered-ports.go

```diff
@@ -822,7 +822,7 @@ UDP         - Unspecified
 https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Registered_ports`,
 	},
 	{
-		Name: "1560–1590",
+		Name: "1560-1590",
```

Comment:
From the commit message:

> The middle dash character "-" was not the ASCII 0xD2 but some other representation: 0xE28093 preventing the correct comparison in the code logic and ignoring the record

<img width="794" alt="non-ascii-char" src="https://user-images.githubusercontent.com/993186/90310335-ed1f7300-def0-11ea-83eb-307a7b5dbb6e.png">

Reply:
The test failing before doing the replacements of chars in files:

```
=== RUN   TestPortsConsistencyValidation
    httpref_test.go:96: Invalid port format: 71–74
    httpref_test.go:96: Invalid port format: 225–241
    httpref_test.go:96: Invalid port format: 249–255
    httpref_test.go:96: Invalid port format: 1011–1020
    httpref_test.go:96: Invalid port format: 1560–1590
    httpref_test.go:96: Invalid port format: 1582–1583
    httpref_test.go:96: Invalid port format: 2222–2226
    [... redacted... 48 offending rules in total]
```

Reply:
Aaaaand this is why the previous test was necessary. :smile: